spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -127,63 +128,64 @@ This specification addresses the need to simplify:
 * Development of server applications that conform both to basic SOAP and
 WSDL standards.
 * Building Web Services that can be deployed with the core Web Services
-APIs and existing J2EE standards.
+APIs and existing Jakarta EE standards.
 * Separate control of public Web Service message contracts and private
 implementation signatures, because in practice public and private
 formats evolve on different schedules.
 
 It is not a goal of this specification to support every feature or to
 enable the creation of every Web Service that it is possible to write
-within existing specifications (JAX-WS 2.0 [5] provides finer control
+within existing specifications (Jakarta XML Web Services 2.3 [5] provides finer control
```

Comment:
avoid references to concrete spec version (just remove `2.3` from here)

Reply:
ok and other places too (https://github.com/eclipse-ee4j/jws-api/pull/69/commits/3d54eeebc5474552c7a719f3932076709eb58551)